### PR TITLE
Add tracking of download clicks for the goodies page

### LINKED ISSUE #973: Google Analytics outbound click tracking for Goodies page downloads
The Goodies page has links to .zip and .pdf files for people to download. Although the hits are probably recorded for a short time in the web server logs, it would be good to track downloads as a proper metric in Google Analytics.

Outbound click listeners on the links could easily track download requests, and record them as Google Analytics events (or even pageviews??)

### PR DESCRIPTION
**Pull request by jacksonj04**
Closes #973.

### CODE REVIEW

File: templates/web/fixmystreet.com/static/posters.html [hunk 1]
```diff
@@ -45,7 +55,7 @@ <h2 id="activist-pack">The FMS Activist Pack</h2>
     <li>Ideas for letting more people know about FixMyStreet</li>
 </ul>
 
-<a href="/cobrands/fixmystreet/posters/activist-pack.zip" class="download-button">Download ZIP file</a>
+<a href="/cobrands/fixmystreet/posters/activist-pack.zip" onclick="trackGoodieDownload('activist-pack', '/cobrands/fixmystreet/posters/activist-pack.zip'); return false;" class="download-button">Download ZIP file</a>
```
Comment (zarino) [inline templates/web/fixmystreet.com/static/posters.html]: Could you just use `$(this).attr('href')` inside `trackGoodieDownload()` to save writing the URL out twice?

Comment (zarino) [inline templates/web/fixmystreet.com/static/posters.html]: That would obviously require you to bind with jQuery though.

**Comment by jacksonj04** [comment]
@zarino Any further comments on this, or is it good to go?

**Comment by zarino** [comment]
@jacksonj04: Nope, it looks ace to me! :+1:

**Comment by dracos** [comment]
For what it's worth, I would have squashed these two commits before merging.

**Comment by dracos** [comment]
And I would have suggested fixing #987 given it was the same lines being edited.

**Pull request merged**